.nvmrc

```diff
@@ -0,0 +1 @@
+v20.18.1
```

Comment:
Version 20 is enough, no improvement into specify the minor too.

```suggestion
v20
```

Reply:
Maybe we can use v22?

Reply:
+1 for 22